### ISSUE TITLE
SystemLocale and IEEnhancedSecurityConfiguration are missing from version 7.1.0.0

### ISSUE THREAD

**Issue by fullenw1** [dsccommunity/ComputerManagementDsc#308]
#### Details of the scenario you tried and the problem that is occurring
Version 7.1.0.0 is supposed to have the new **SystemLocale** and **IEEnhancedSecurityConfiguration** resources.
However they have not been implemented.
#### Verbose logs showing the problem
```powershell
PS> Get-DscResource -Module ComputerManagementDsc | Where-Object -Property Version -EQ '7.1.0.0'

ImplementedAs   Name                      ModuleName                     Version    Properties                                        
-------------   ----                      ----------                     -------    ----------                                        
PowerShell      Computer                  ComputerManagementDsc          7.1.0.0    {Name, Credential, DependsOn, Description...}     
PowerShell      OfflineDomainJoin         ComputerManagementDsc          7.1.0.0    {IsSingleInstance, RequestFile, DependsOn, PsDs...
PowerShell      PendingReboot             ComputerManagementDsc          7.1.0.0    {Name, DependsOn, PsDscRunAsCredential, SkipCcm...
PowerShell      PowerPlan                 ComputerManagementDsc          7.1.0.0    {IsSingleInstance, Name, DependsOn, PsDscRunAsC...
PowerShell      PowerShellExecutionPolicy ComputerManagementDsc          7.1.0.0    {ExecutionPolicy, ExecutionPolicyScope, Depends...
PowerShell      RemoteDesktopAdmin        ComputerManagementDsc          7.1.0.0    {IsSingleInstance, DependsOn, Ensure, PsDscRunA...
PowerShell      ScheduledTask             ComputerManagementDsc          7.1.0.0    {TaskName, ActionArguments, ActionExecutable, A...
PowerShell      SmbServerConfiguration    ComputerManagementDsc          7.1.0.0    {IsSingleInstance, AnnounceComment, AnnounceSer...
PowerShell      SmbShare                  ComputerManagementDsc          7.1.0.0    {Name, Path, CachingMode, ChangeAccess...}        
PowerShell      TimeZone                  ComputerManagementDsc          7.1.0.0    {IsSingleInstance, TimeZone, DependsOn, PsDscRu...
PowerShell      VirtualMemory             ComputerManagementDsc          7.1.0.0    {Drive, Type, DependsOn, InitialSize...}          
PowerShell      WindowsCapability         ComputerManagementDsc          7.1.0.0    {Name, DependsOn, Ensure, LogLevel...}            
PowerShell      WindowsEventLog           ComputerManagementDsc          7.1.0.0    {LogName, DependsOn, IsEnabled, LogFilePath...}

PS>
```

#### Suggested solution to the issue
Re-publish the module available in PSGallery and include the missing resources.

#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
N/A

#### The operating system the target node is running

N/A

#### Version and build of PowerShell the target node is running

N/A

#### Version of the DSC module that was used ('dev' if using current dev branch)
7.1.0.0

**Comment by PlagueHO** [comment]
Hi @fullenw1 - these resources aren't in v7.1.0.0. You can find them in v8.0.0-preview0001 and they'll be released as a final release in v8.0.0 in the next few days, once https://github.com/dsccommunity/ComputerManagementDsc/pull/304 is merged.

**Comment by fullenw1** [comment]
Ok I see! The README.md file was a bit in advance with the resource list :)
I have checked v8.0.0 which has been released since and it contains the formerly missing resources.
Thanks @PlagueHO  !
